stores/customer/store.go

```diff
@@ -0,0 +1,48 @@
+package customer
+
+import (
+	"customerApi/models"
+	"customerApi/stores"
+	"database/sql"
+)
+
+type store struct {
+	db *sql.DB
+}
+
+func New(db *sql.DB) stores.Customer {
+	return store{db: db}
+}
+
+func (s store) CreateCustomer(c models.Customer) error {
```

Comment:
`customer.CreateCustomer` looks redundant. `customer.Create` should be fine.

Reply:
done